package.json

```diff
@@ -47,7 +47,7 @@
     "object-assign": "^4.1.0"
   },
   "devDependencies": {
-    "ava": "0.16.0",
+    "ava": "^1.0.0-beta.6",
```

Comment:
Any change we can use the [latest stable](http://npm.im/ava) (0.25.0) here instead of the beta?

Reply:
Sure, we can. It was an optional fix, which improves the visibility of the diff in case of error during testing. They really improved the diff a lot in last versions. I'll get rid of this bump then.